holoviews/plotting/plotly/renderer.py

```diff
@@ -3,12 +3,35 @@
 import base64
 
 import param
+import panel as pn
+
+from param.parameterized import bothmethod
+
 with param.logging_level('CRITICAL'):
     import plotly.graph_objs as go
 
 from ..renderer import Renderer, MIME_TYPES, HTML_TAGS
 from ...core.options import Store
 from ...core import HoloMap
+from .callbacks import callbacks
+
+
+def _PlotlyHoloviews(fig_dict):
```

Comment:
I find the naming convention here confusing, it is a `def` and therefore a function, but it is capitalized like a class. It is also underscored which is not a convention generally used in HoloViews for classes. In addition, a more descriptive name would be good...

Reply:
I think we should be underscoring more things that are completely private and shouldn't be used in any other context, so I vote to keep the underscore. I'll rename to ``_PlotlyHoloviewsPane`` and add a docstring.

Reply:
The underscore seems fine, but why is it capitalized?

Reply:
Because it slightly modifies the behavior of the regular `HoloViews` pane.

Reply:
i.e. it is an alternative class constructor.

Reply:
Even if you keep the underscore and capitalization, I still think a more meaningful name would be good.

Reply:
Thats why I added Pane to the name.